[BIM-2427] Adds documentation for the new GUI namespace in the bim viewer. (#151)
procore/documentation: bim_webviewer/bim_web_viewer.md

Promise(boolean)
```

## GUI Namespace

<p class="heading-link-container"><a class="heading-link" href="#gui-namespace"></a></p>

### Add Context Menu Item

<p class="heading-link-container"><a class="heading-link" href="#add-context-menu-item"></a></p>

```js
addContextMenuItem({label, id, shortcut, onClick, singleObject});
```

#### Description

Retrieves a JavaScript object that describes the object.

#### Parameters

| Field Name | Required | Type    | Description                                               |
|-----------| --- |---------|-----------------------------------------------------------|
| label     | true | String  | Human-readable label of the new contet menu entry         |
| id        | true | String  | Unique identifier for the new context menu entry          |
| shortcut  | true | char[]  | Keyboard-shortcut equivalent                              |
| onClick   | true | callback | JS function triggered when menu entry is clicked          |

> | singleObject   | true | boolean | Indicates if the action applies only to single selections |

Reviewer: Lets remove the white space

> | Field Name | Required | Type    | Description                                               |
> |-----------| --- |---------|-----------------------------------------------------------|
> | label     | true | String  | Human-readable label of the new contet menu entry         |
> | id        | true | String  | Unique identifier for the new context menu entry          |
> | shortcut  | true | char[]  | Keyboard-shortcut equivalent                              |
> | onClick   | true | callback | JS function triggered when menu entry is clicked          |
> | singleObject   | true | boolean | Indicates if the action applies only to single selections |

Reviewer 2: ```suggestion
| Field Name | Required | Type | Description |
| - | - | - | - |
| label | true | String | Human-readable label of the new context menu entry |
| id | true | String  | Unique identifier for the new context menu entry |
| shortcut  | true | char[]  | The keyboard-shortcut label to show. Does not register a keyboard listener |
| onClick | true | callback | JS function triggered when menu entry is clicked |
| singleObject | true | boolean | Indicates if the action applies only to single selections |
```

##### Returns

HTMLDivElement

##### Namespace

GUI

---

### Remove Context Menu Items

<p class="heading-link-container"><a class="heading-link" href="#remove-context-menu-items"></a></p>

```js
removeContextMenuItems({contextMenuItemIds});
```

#### Description

Retrieves a JavaScript object that describes the object.

#### Parameters

> | Field Name | Required | Type     | Description                                  |
> |-----------| --- |----------|----------------------------------------------|
> | contextMenuItemIds     | true | String[] | Array of the Context Menu Item Ids to remove |

Reviewer 2: ```suggestion
| Field Name | Required | Type | Description |
| - | - | - | - |
| contextMenuItemIds | true | String[] | Array of the Context Menu Item Ids to remove |
```
